### PR TITLE
feat: telekom-mms.grafana role

### LINKED ISSUE #340: Merge `telekom-mms.grafana` into `community.grafana`
##### SUMMARY

Merge [telekom-mms.grafana](https://github.com/telekom-mms/ansible-role-grafana) in `community.grafana`

##### ISSUE TYPE
- Feature Idea

##### COMPONENT NAME

- collection roles
##### ADDITIONAL INFORMATION

As an addition to #322, I would like to suggest merging the role `telekom-mms.grafana` into the `community.grafana` collection.
@rrey @seuf What are your thoughts about that?

### PR DESCRIPTION
**Pull request by Nemental**
##### SUMMARY

Resolves #340 

##### ISSUE TYPE

- Feature Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: changelogs/fragments/343-telekom-mms-role.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+major_changes:
```
Comment (rndmh3ro) [inline changelogs/fragments/343-telekom-mms-role.yml:1]: I'm not sure this is a major-change, but rather a minor.

https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment

What do you think?

File: roles/grafana/CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# Changelog
```
Comment (rndmh3ro) [inline roles/grafana/CHANGELOG.md:1]: How should we treat this changelog in the future? A separate CL apart from the collection is undesirable.

**Review by rndmh3ro — COMMENTED** [review]
Awesome work!

I've left two questions to ponder.

File: changelogs/fragments/343-telekom-mms-role.yml (hunk 1 above)
Comment (Nemental) [reply changelogs/fragments/343-telekom-mms-role.yml:1]: You're right - now that I think about it, it's not a major change... Adding the role doesn't affect existing collection code or user code.

File: roles/grafana/CHANGELOG.md (hunk 2 above)
Comment (Nemental) [reply roles/grafana/CHANGELOG.md:1]: I've already thought about removing it. There's nothing special to be found there...

**Review by rndmh3ro — APPROVED** [review]

**Pull request merged**